src/terraform.js

```diff
@@ -56,7 +56,7 @@ const iterativeProviderTpl = () => {
 terraform {
   required_providers {
     iterative = {
-      source = "iterative/iterative"
+      source = "github.com/iterative/iterative"
```

Comment:
@dacbd is this intended or maybe you forgot to change it back maybe?

Reply:
Intended, I'll rebase the first commit out or change it back before it's done/marked as ready for review.